src/_repobee/cli/preparser.py

```diff
@@ -46,6 +46,26 @@ def parse_args(
         prog="repobee", description="plugin pre-parser for _repobee."
     )
 
+    add_arguments(parser, default_config_file)
+    args = parser.parse_args(sys_args)
+
+    return args
+
+
+def add_arguments(
+    parser: argparse.ArgumentParser,
+    default_config_file: Optional[pathlib.Path] = None,
+) -> None:
+    """Add argument flags that the preparser handles to the parser. When
+    called for the preprocessor, the second argument should be defined,
+    and when called from the main parser, it can be ignored as the argument
+    will be deleted from the parsed args.
```

Comment:
Setting a default value for `default_config_file` is kind of asking for a mistake down the line. Ideally, in my opinion, a function shouldn't need to be concerned with what context it's called in (here, called for preparser or mainparser), neither in implementation nor in docs.

I think it'd be better to remove the default argument, and explicitly call it with `default_config_file = None` for the mainparser. What do you think?

Reply:
For clarification, you would like to see in the mainparser: `preparser.add_arguments(parser, default_config_file=None)` or is `preparser.add_arguments(parser, None)` fine? (asking as I would lean towards the latter, but not sure if you wanted the former).

Reply:
Oh, I the style of the invocation doesn't matter. I just thought it would be better to pass the appropriate argument, rather than to have it implicit in the function call.

Reply:
Alright, done!